main.ts

```diff
@@ -263,14 +283,18 @@ export default class ReadwiseMirror extends Plugin {
           last_highlight_at: last_highlight_at ? this.formatDate(last_highlight_at) : '',
           source_url: source_url,
           tags: this.formatTags(tags),
+          highlight_tags: this.formatTags(highlightTags),
+          tags_nohash: this.formatTags(tags, true),
+          hl_tags_nohash: this.formatTags(highlightTags, true),
         };

```

Comment:
After further reflection and looking into what makes valid frontmatter tags, I realized the "quote" approach was the wrong one, and the correct way to deal with tags in frontmatter would be to strip them of the hashtag (#). The `obsidian-linter` plugin e.g. moves tags from the body to frontmatter and removes the hashtag (#) and doesn't add any quotes either. 

I've updated the change accordingly and also in the Readme.

Reply:
After further work with the plugin, I realized some edge-cases, namely tags with "@" in it, are invalid in Obsidian frontmatter, unless the tags has single or double quotes.

I modified the plugin accordingly:

- when we strip the "#", we add a single quote "'"